src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -83,7 +84,13 @@ def __init__(
         :param interface: python-can interface modules used
         :param channel: the can interface name
         :param state: BusState of the channel
-        :param trace_path: path to write the trace
+        :param trace_path: path to write the trace (can be a folder or a .trc file)
```

Comment:
This description is super long maybe add a ..note:: or ..warning:: tags and have it  a little bit more highlighted in the documentation

Reply:
Done